backend/src/main/java/com/example/skicurort/curort/CurortControler.java

```diff
@@ -0,0 +1,63 @@
+package com.example.skicurort.curort;
+
+import com.example.skicurort.exception.Error;
+import com.example.skicurort.exception.NoIdException;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/curort")
+public class CurortControler {
+
+  private final CurortService curortService;
+
+  @GetMapping("/")
+  ResponseEntity<List<CurortDTO>> getAll() {
+    return ResponseEntity.ok(curortService.getAll());
+  }
+
+  @PostMapping("/")
+  public ResponseEntity<CurortDTO> save(@RequestBody CurortDTO curortDto) {
+
+    curortService.save(curortDto);
+    return ResponseEntity.ok(curortDto);
```

Comment:
tutaj dobrze byłoby zwrócić DTO zmapowany z zapisanej encji - wtedy klient dostanie DTO z przypisanym ID

Reply:
czy mógłbym prosić o podpowiedz jak to zrobić?

Reply:
`return ResponseEntity.ok(curortService.save(curortDto));`